x/validator-vesting/querier.go

```diff
@@ -1,358 +0,0 @@
-package validator_vesting
```

Comment:
Just noting that I believe coinmarketcap does use these.  I'm not sure a grpc querier would let us return an int value as the response